exporter/containerimage/export.go

```diff
@@ -115,28 +116,31 @@ func (e *imageExporterInstance) Export(ctx context.Context, src exporter.Source)
 	}()
 
 	if e.targetName != "" {
-		if e.opt.Images != nil {
-			tagDone := oneOffProgress(ctx, "naming to "+e.targetName)
-			img := images.Image{
-				Name:      e.targetName,
-				Target:    *desc,
-				CreatedAt: time.Now(),
-			}
-
-			if _, err := e.opt.Images.Update(ctx, img); err != nil {
-				if !errdefs.IsNotFound(err) {
-					return nil, tagDone(err)
+		targetNames := strings.Split(e.targetName, ",")
+		for _, targetName := range targetNames {
+			if e.opt.Images != nil {
```

Comment:
nit: `if e.opt.Images != nil { targetNames := ...`?

Reply:
I was not 100% sure about the semantic of that and therefore I tried to keep the change as small as possible (I effectively just moved the old code in a loop)